### PR TITLE
refactor: intro screens text content

### PR DESCRIPTION
**Pull request by JeremiGendron**
```
fix(intro): Your Duty (8, 9, 10)
fix(intro): Your Responsibility (11)
fix(intro): Your Turn (12)
fix(intro): Reformat Your responsibility to fit more text
```
Implement suggestions from justin on intro screen texts.

Ref: #891 
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by alexbarnsley** [comment]
I've passed this on to @boldninja 	to take a look and have a read

File: src/renderer/i18n/locales/en-US.js [hunk 1]
```diff
@@ -48,29 +48,30 @@ export default {
...
     },
     RESPONSIBILITY: {
       TITLE: 'Responsibility',
-      STORAGE: 'You have the option of storing an encrypted copy of your passphrase locally. Please remember, your passphrase IS your account. If you choose this option and your computer dies, your account is lost forever.',
-      BACKUP: 'Always backup your passphrase and keep it in a safe place. The best practice is to write it down on thick paper and store it in a fireproof safe or save it on an encrypted flash drive.',
-      PASSPHRASE: 'Remember, anyone who has your passphrase can access your money. Never share your account with anyone and avoid storing it in accounts that are susceptible to hacking, such as cloud services.'
+      STORAGE: 'The passphrase of each wallet address is able to sign transactions and move funds. This means if your computer dies but you have your passphrase, you can still access your funds. The ARK Desktop Wallet has an additional feature that lets you set an encrypted password as well, for easier management. If your computer dies, you cannot use the encrypted password to access your funds from a different machine. You will need the passphrase.',
```
[the remark this answers is not in the document]
Comment (j-a-m-l) [reply src/renderer/i18n/locales/en-US.js]: You can use the `i18n` functional component (https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage), or split it up into multiple segments, as you prefer.

[the remark this answers is not in the document]
Comment (JeremiGendron) [reply src/renderer/i18n/locales/en-US.js]: thanks! will do that today.

[the remark this answers is not in the document]
Comment (JeremiGendron) [reply src/renderer/i18n/locales/en-US.js]: Wasn't sure whether to make bold all the occurrences, so only touched up the first paragraph as you've outlined.

https://github.com/ArkEcosystem/desktop-wallet/commit/04e3558d4d92f70e21fc53dfd4b52ee77eac3092#diff-92265712db23686ca615a58f84db7f52R72

**Pull request merged**